src/applications/vaos/components/ConfirmedAppointmentListItem.jsx

```diff
@@ -97,8 +98,37 @@ export default function ConfirmedAppointmentListItem({
         )}
       </div>
 
+      {!allowCancel &&
+        !canceled && (
+          <div className="vads-u-margin-top--2">
+            <button
+              onClick={() => exportICSFile(appointment, facility)}
+              aria-label="App appointment to calendar"
```

Comment:
I would guess you mean "Add" instead of "App"?

Reply:
Done